libs/php-commons/XSSHelpers.php

```diff
@@ -61,6 +61,7 @@ public static function xscrub($data)
      */
     public static function sanitizer($in, $allowedTags = array(), $encoding = 'UTF-8')
     {
+        $errorReportingLevel = error_reporting(0);//turn off errors for the next few lines
```

Comment:
It might be a little better to use:

```
error_reporting(E_ALL ^ E_WARNING); 
```

instead, since it's only the warnings that should be suppressed.